commons/src/types/ScheduleEditor.ts

```diff
@@ -30,9 +30,17 @@ export interface Manifest extends NylasManifest {
   date_format: "full" | "weekday" | "date" | "none";
   open_hours: AvailabilityRule[];
   recurrence: "none" | "required" | "optional";
-  recurrence_cadence: ("none" | "daily" | "weekdays" | "weekly" | "biweekly" | "monthly")[];
+  recurrence_cadence: (
+    | "none"
+    | "daily"
+    | "weekdays"
+    | "weekly"
+    | "biweekly"
+    | "monthly"
+  )[];
   overbooked_threshold: number;
   mandate_top_of_hour: boolean;
   show_preview: boolean;
   timezone: string;
+  screen_bookings: boolean;
```

Comment:
I think this could be clearer. What about `require_event_confirmation` or `require_host_confirmation`?

Reply:
Going to merge as it stands right now; I've got a conversation with @hazik and @chantalala going about nomenclature that we can revisit later.